tests/config/test_data.config

```diff
@@ -184,6 +184,7 @@ params {
                 genome_21_sdf                                  = "${params.test_data_base}/data/genomics/homo_sapiens/genome/chr21/sequence/genome_sdf.tar.gz"
                 genome_21_fasta                                = "${params.test_data_base}/data/genomics/homo_sapiens/genome/chr21/sequence/genome.fasta"
                 genome_21_fasta_fai                            = "${params.test_data_base}/data/genomics/homo_sapiens/genome/chr21/sequence/genome.fasta.fai"
+                genome_21_gtf                                  = "${params.test_data_base}/data/genomics/homo_sapiens/genome/chr21/sequence/chr21_gencode.gtf"
```

Comment:
I think you need to name this explicitly gencode, also is there anything else different about this gtf file?

Reply:
nah its just gencode chr21 - renamed now!